doc/salobj_cscs.rst

```diff
@@ -101,7 +101,7 @@ Most CSCs can be configured.
       * ``setValue``. This is strongly discouraged, for reasons given below.
```

Comment:
I think this `CSC Details` would benefit from some reformatting. The information there is very dense and does't present in a good way to the readers. 

I think you could make each of those major bullet points into subsections with an explanatory title and some support text. I would also like to see an example of how to issue the acknowledgement. I am sure I could figure out by doing some digging around based on the information here but I think it would be better if **all** the information was present here.

Reply:
We might want to make that a separate ticket.

Reply:
agreed....